public/app/plugins/datasource/prometheus/mode-prometheus.js

```diff
@@ -46,7 +46,7 @@ var PrometheusHighlightRules = function() {
       regex : "[a-zA-Z_$][a-zA-Z0-9_$]*\\b"
     }, {
       token : "keyword.operator",
-      regex : "\\+|\\-|\\*|\\/|%|\\^|=|==|!=|<=|>=|<|>|=\\~|!\\~"
+      regex : "\\+|\\-|\\*|\\/|%|\\^|==|!=|<=|>=|<|>"
```

Comment:
label match operator is added to another `keyword.operator` token. no need to add here.